porchlightapi/settings.py

```diff
@@ -35,5 +36,14 @@
                                'PORCHLIGHT_JSON_FILE',
                                PORCHLIGHT_JSON_FILE_DEFAULT)
 
+PORCHLIGHT_GITHUB_AUTH_DEFAULT = None
+PORCHLIGHT_GITHUB_AUTH = getattr(settings,
+                                 'PORCHLIGHT_GITHUB_AUTH',
+                                 PORCHLIGHT_GITHUB_AUTH_DEFAULT)
+
+PORCHLIGHT_GITHUB_TAG_PATTERN_DEFAULT = r'^v[.0-9]+'
```

Comment:
@willbarton To your question about whether a tagged repo can be treated as a deployment, I'd flip that on its head and say the question is: "How do you know when a commit is deployed?"

To me, this answer will vary among teams or projects and consequently is configurable. This feels a lot like a problem cleanly solved with a Strategy pattern.

In our case at work right now, it's a good-enough assumption that a tag of the pattern you describe will be closely followed by a deploy. There might be a day or two lag, but nothing that would junk up the data for porchlight's purposes

Certainly we can all imagine other better strategies that accurately answer the "when is this commit deployed?" question. But I think at least for our purposes now, our job here is to create an architecture that allows for pluggable strategies but leaves the implementation of those more accurate strategies up in the air, until such a time as we need it or another interested party contributes it. 

That said, if you or others have ideas about how we can start collecting that data right now, let's get that rolling. We actually know all this information, but it's currently not quite so clean to get to. Let's get any ideas we all have about how to simplify storing or accessing that information out of our heads and documented. Might be better to do that internally though so that we have full freedom to discuss our architecture and infrastructure without inadvertently disclosing information we shouldn't.